src/components/NurseryWithdrawals/NurseryWithdrawalsDetails.tsx

```diff
@@ -1,13 +1,193 @@
+    color: theme.palette.TwClrTxtBrand,
+    alignItems: 'center',
+    marginLeft: 0,
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const TABS = ['withdrawal', 'reassignment'];
 
 type NurseryWithdrawalsDetailsProps = {
   organization: ServerOrganization;
 };
 
-export default function NurseryWithdrawalsDetails(props: NurseryWithdrawalsDetailsProps): JSX.Element {
+export default function NurseryWithdrawalsDetails({ organization }: NurseryWithdrawalsDetailsProps): JSX.Element {
+  const classes = useStyles();
+  const theme = useTheme();
   const { withdrawalId } = useParams<{ withdrawalId: string }>();
+  const { isMobile } = useDeviceInfo();
+  const contentRef = useRef(null);
+  const [snackbar] = useState(useSnackbar());
+
+  const query = useQuery();
+  const history = useHistory();
+  const location = useStateLocation();
+  const tab = (query.get('tab') || '').toLowerCase();
+  const preselectedTab = TABS.indexOf(tab) === -1 ? 'withdrawal' : tab;
+  const [selectedTab, setSelectedTab] = useState(preselectedTab);
+
+  const [withdrawal, setWithdrawal] = useState<NurseryWithdrawal | undefined>(undefined);
+  const [delivery, setDelivery] = useState<Delivery | undefined>(undefined);
+  const [batches, setBatches] = useState<Batch[] | undefined>(undefined);
+  useEffect(() => {
+    const updateWithdrawal = async () => {
+      const withdrawalResponse = await getNurseryWithdrawal(withdrawalId);
+      if (!withdrawalResponse.requestSucceeded || withdrawalResponse.error) {
+        setWithdrawal(undefined);
+        setDelivery(undefined);
+        setBatches(undefined);
+        snackbar.toastError(`An Error Occurred: ${withdrawalResponse.error}`);
+      } else {
+        setWithdrawal(withdrawalResponse.withdrawal);
+        setDelivery(withdrawalResponse.delivery);
+        setBatches(withdrawalResponse.batches);
+      }
+    };
+
+    updateWithdrawal();
+  }, [withdrawalId, snackbar]);
+
+  useEffect(() => {
+    setSelectedTab((query.get('tab') || 'withdrawal') as string);
+  }, [query]);
+
+  const tabStyles = {
+    fontSize: '14px',
+    padding: theme.spacing(1, 2),
+    minHeight: theme.spacing(4.5),
+    textTransform: 'capitalize',
+    '&.Mui-selected': {
+      color: theme.palette.TwClrTxtBrand as string,
+      fontWeight: 500,
+    },
+  };
+
+  const contentPanelProps = {
+    borderRadius: '32px',
+    backgroundColor: theme.palette.TwClrBg,
+    padding: theme.spacing(3),
+  };
+
+  const handleTabChange = (newValue: string) => {
+    query.set('tab', newValue);
+    history.push(getLocation(location.pathname, location, query.toString()));
+  };
 
-  return <TfMain>Nursery Withdrawal Details {withdrawalId} </TfMain>;
+  return (
+    <TfMain>
+      <PageHeaderWrapper nextElement={contentRef.current} nextElementInitialMargin={-24}>
+        <Box marginBottom={theme.spacing(4)}>
+          <Box>
+            <Link id='back' to={APP_PATHS.NURSERY_WITHDRAWALS} className={classes.backToWithdrawals}>
```

Comment:
you can use Link component from `components/common/Link` - you may need to rebase your branch to get latest code
speaking of which,, I missed updating the back to links on all views..

Reply:
I didn't update the back links because I thought they were different from other links

Reply:
good point, @chudo76 do the back links need to be updated as well? (underline + 500 weight)

Reply:
> good point, @chudo76 do the back links need to be updated as well? (underline + 500 weight)

Yes, all links styles going forward will be:

- Links are `tw-clr-txt-brand`, no text decoration.
- On hover, `text-decoration: underline;` 
- No additional styles for `:active` or `:visited`.

Reply:
Thanks @chudo76 i will open up a new ticket out-of-band of withdrawal log effort

Reply:
+1 thanks for responding!